src/popup-generation/FieldInputRow.js

```diff
@@ -5,19 +5,14 @@ import {FIELD_TYPES} from "./constants";
 const calculateUIInfo = (fieldType) => {
 	switch(fieldType) {
 		case FIELD_TYPES.text:
+		case FIELD_TYPES.number:
 			return {
-				showFieldName: true,
 				fieldDisplayNameLabel: 'Field Display Name',
 			}
-		case FIELD_TYPES.number:
+		case FIELD_TYPES.link:
 			return {
-				showFieldName: true,
-				fieldDisplayNameLabel: 'Field Display Name',
+				fieldDisplayNameLabel: 'Link Text',
 			}
-		case FIELD_TYPES.zillowLink:
-			return {}
-		case FIELD_TYPES.wikipediaLink:
-			return {}
 		case FIELD_TYPES.subSectionHeader:
```

Comment:
There's a bit of noise here, since I improved the code by
1. removing the `showFieldName` bool and replacing it with more explicit checks in the Component body
2. Had type `text` fall through to the identical type `number`

You can ignore that. Just focus on the fact I dropped zillow/wikipedia and added link.